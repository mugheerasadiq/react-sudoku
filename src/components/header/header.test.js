import React from "react";
import renderer from "react-test-renderer";

import Header from "./Header.index";

it("All props are passed", () => {
  const tree = renderer.create(<Header />).toJSON();
  expect(tree).toMatchSnapshot();
});
