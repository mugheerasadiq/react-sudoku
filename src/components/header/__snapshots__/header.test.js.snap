// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`All props are passed 1`] = `
<div
  className="header_container"
>
  <h1
    style={
      Object {
        "color": "cadetblue",
        "fontSize": "42px",
        "fontWeight": "bolder",
      }
    }
  >
    suGOku
  </h1>
  <img
    height={150}
    src="logo.jpg"
    width={150}
  />
</div>
`;
